src/plugins/files/server/routes/file_kind/integration_tests/file_kind_http.test.ts

```diff
@@ -72,7 +72,7 @@ describe('File kind HTTP API', () => {
       .expect(200);
 
     expect(header['content-type']).toEqual('image/png');
-    expect(header['content-disposition']).toEqual('attachment; filename="test.png"');
+    expect(header['content-disposition']).toEqual('attachment; filename=test.png');
```

Comment:
The `res.file` implementation provides the filename for content-disposition header without quotes, and is still a valid value, see [the spec](https://httpwg.org/specs/rfc6266.html#examples)